### PR TITLE
modify the way of getting GOPATH variable

### PR DESCRIPTION
**Pull request by gy95**
Signed-off-by: gy95 <guoyao17@huawei.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
we can get GOPATH by `go env GOPATH`, and just only once
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: hack/karmada-bootstrap.sh [hunk 1]
```diff
@@ -124,7 +124,7 @@ export REGISTRY="swr.ap-southeast-1.myhuaweicloud.com/karmada"
 make images
 
 GO111MODULE=on go install "github.com/karmada-io/karmada/cmd/karmadactl"
-GOPATH=$(go env | grep GOPATH | awk -F '=' '{print $2}'| sed 's/\"//g')
```
Comment (RainbowMango) [inline hack/karmada-bootstrap.sh:127]: The GOPATH is a path list, we just need the first path.

According to `go help gopath`:
```
The GOPATH environment variable lists places to look for Go code.
On Unix, the value is a colon-separated string.
On Windows, the value is a semicolon-separated string.
On Plan 9, the value is a list.
```

Comment (gy95) [reply hack/karmada-bootstrap.sh:127]: when i set `GOPATH=/root/go:/home/go`, and run `go env | grep GOPATH | awk -F '=' '{print $2}'| sed 's/\"//g'`, the output is `/root/go:/home/go`, we should find another way to split gopath list to select the first path

Comment (futuretea) [reply hack/karmada-bootstrap.sh:127]: go env GOPATH | awk -F ':' '{print $1}'

Comment (gy95) [reply hack/karmada-bootstrap.sh:127]: that's it, considering different os, we can use `go env GOPATH | awk -F ':|;' '{print $1}'`, but i dont know On Plan 9, what is GOPATH like

**Pull request merged**